### DISCUSSION TITLE
"'mysql' is not recognized as an internal or external command,

### DISCUSSION THREAD

**Discussion by patolax** [Daniel-Yonkov/jest-mysql#83] [General]
This is a question. 

Do I need mysql db installed on my local machine to run the tests? I get above error even after I install mysql and jest packages.

My database is on aws, can aws aurora mysql be used as a test Database?

I am using mysql2, can this package be used with mysql2 simultaneously?

**Comment by Daniel-Yonkov** [comment]
Hello @patolax - This error probably appears when you have a db scheme that you want to setup before running your tests. In order to archive that, you would need the **mysql client** and if such is missing, this error appears. 
If you are on NIX based distro (linux/mac) the following command should print the path to your mysql client if installed:
```sh
which mysql
# example output
# /usr/bin/mysql
```
If this does not resolve in a path, then you should install a mysql client on your machine (the client is enough, it's not needed to have a database engine itself).

In regards of the database setup - it should work as long as you have a **mysql client** to connect to a remote database and the remote database is accessible from outside the local network.

There is a suggestion in regards of implementing the support for mysql 2 in discussion #79 where you can find a bit more details. Currently there is no plan to implement support for mysql2 but if more interest appears towards the library I may consider supporting it.
